utils/errorMessage.js

```diff
@@ -43,6 +43,9 @@ const errorMessage = {
   invalidPlan: {
     message: 'The plan you requested does not exist.',
   },
+  tooManyUsers: {
+    message: 'There are too many staff accounts on this hotel to switch plans.',
```

Comment:
I'd rephrase this so it's more descriptive. Something like: 
`The number of staff accounts on this hotel is too high for selected plan.`